ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -353,7 +389,7 @@ class GraphsContainer extends React.Component {
       bottom: 50,
     };
 
-    const today = moment.utc().toDate();
+    const domainX = this.getDomainX();
```

Comment:
There's still a bug with this implementation.

When the user is 1st landed to the Graphs view page, the big graph from the bottom correctly display the dots from the edges.
However, if the user clicks the upper graph, the zoom of the bottom graph refreshes & comes back to the old behavior, cutting (again) off the 1st point from the left.

We need to address this issue as well.

Reply:
Ah, nice catch! I didn't adjust the domain of the Brush component. I re-worked my new code to fit in with the existing `initZoomDomain` code.